service/.classpath

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
i think this file can be removed, maybe add it to `.gitignore`? Something like *.classpath